zio-http-logging/src/test/scala/zhttp/logging/LogLevelSpec.scala

```diff
@@ -15,5 +15,10 @@ object LogLevelSpec extends DefaultRunnableSpec {
         assertTrue(LogLevel.fromString(level.toString) == level)
       }
     },
+    testM("any value with the exception of defined values for LogLevel should be set to Error log level.") {
+      checkAll(Gen.fromIterable(List("not defined", "unknown", "disable"))) { level =>
```

Comment:
```scala
Gen.anyString
```

This would be better.

Reply:
But it might fail if Gen will produce Info, Error or any defined log levels. Probability is small but still it might fail.